### PR TITLE
{lib}[GCCcore/11.2.0] Mako v1.1.4, LLVM v12.0.1, libdrm v2.4.107, ... w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/3bb7aa20fc84c6ec841f7f6c5ee89ecc for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in total)
bear-pg0305u19a.bear.cluster - Linux RHEL 8.4, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/b125aa5ed8e96b5384c852e7cb1d18b7 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
bear-pg0305u19a.bear.cluster - Linux RHEL 8.4, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/f398a7266a80bbc459109dcb7fa9338b for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Comment by Micket** [comment]
https://docs.mesa3d.org/relnotes/21.2.0.html using 21.2.0 was a bit of a mistake. Shall I go ahead and switch this back to 21.1.7?

File: easybuild/easyconfigs/m/Mesa/Mesa-21.2.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# - EGL is available
+#
+# Software renderers are enabled based on system architecture:
+# - llvmpipe: the high-performance Gallium LLVM driver
+# - swr: Intel's OpenSWR
+# Default renderer is llvmpipe. SWR can be enabled by setting the environment
+# variable GALLIUM_DRIVER=swr
+
+name = 'Mesa'
+version = '21.2.0'
```
Comment (boegel) [inline easybuild/easyconfigs/m/Mesa/Mesa-21.2.0-GCCcore-11.2.0.eb:13]: Note: Mesa 21.2.0 is not a stable release, so this was moved to 21.1.7 in #13736

**Pull request merged**